roles/collect_modified_images/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# In order for this role to work, add the following task to your playbook
+# immediately before calling the deployment role.
+# - name: "Set the deployment start time"$a
+#   shell: "date -u '+%Y-%m-%dT%H:%M:%SZ'"
+#   register: start_time
+- name: "Create empty list for images"
+  set_fact:
+    pulled_images: []
+    events_file_path: "/tmp/events.txt"
+
+- name: "Generate events.json"
+  shell: "{{ oc_bin_path }} {% raw %}get events --field-selector reason==Pulling -o go-template='{{range .items}}{{.lastTimestamp}},{{.message}}{{\"\\n\"}}{{end}}' --sort-by={.lastTimestamp}{% endraw %}"
```

Comment:
I believe this command should be run with the `--all-namespaces` option to cover all image pulls on the cluster - operators can manage other namespaces than the one they were installed in.